File: 2026POTD/JANUARY/12-01-2026/1266.c
int minTimeToVisitAllPoints(int** points, int pointsSize, int* pointsColSize) {
    int ans = 0;
    for (int i = 1; i < pointsSize; i++) {
        int dx = points[i][0] - points[i - 1][0];
        int dy = points[i][1] - points[i - 1][1];
        if (dx < 0) dx = -dx;
        if (dy < 0) dy = -dy;
        ans += dx > dy ? dx : dy;
    }
    return ans;
}

File: 2026POTD/JANUARY/12-01-2026/1266.php
class Solution {

    /**
     * @param Integer[][] $points
     * @return Integer
     */
    function minTimeToVisitAllPoints($points) {
        $ans = 0;
        for ($i = 1; $i < count($points); $i++) {
            $dx = abs($points[$i][0] - $points[$i - 1][0]);
            $dy = abs($points[$i][1] - $points[$i - 1][1]);
            $ans += max($dx, $dy);
        }
        return $ans;
    }
}